src/components/elements/typography.js

```diff
@@ -25,8 +25,12 @@ export const BaseElement = css`
 export const Text = styled.p`
     ${BaseElement}
     font-weight: ${(props) => props.weight || 'normal'};
-    font-size: ${(props) => props.size || 'var(--text-size-s)'};
+    font-size: ${(props) => props.size || '1.6rem'};
     width: ${(props) => props.width || 'auto'};
+
+    @media ${device.tabletL} {
+        font-size: ${(props) => props.size || '16px'};
+    }
 `
 
 export const Header = styled(({ as = 'h1', children, ...props }) =>
```

Comment:
pls remove type from spread, and add default type 🙏

Reply:
As discussed, this will not be required for the header component